# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glib/no-tests.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glib-2.62.2/gio/meson.build.vanilla	2020-02-01 22:38:00.193564311 +0100
+++ glib-2.62.2/gio/meson.build	2020-02-01 22:48:20.601564612 +0100
@@ -1010,7 +1010,7 @@
 
 subdir('fam')
 if build_tests
-    subdir('tests')
+    #subdir('tests')
 endif
 
 # The following is an example for building internal marshallers that are used
--- glib-2.62.2/meson.build.vanilla	2020-02-01 22:48:00.137564602 +0100
+++ glib-2.62.2/meson.build	2020-02-01 22:48:06.985564606 +0100
@@ -2103,7 +2103,7 @@
 subdir('gio')
 subdir('fuzzing')
 if build_tests
-  subdir('tests')
+  #subdir('tests')
 endif
 
 # xgettext is optional (on Windows for instance)
